board/views.py

```diff
@@ -93,21 +96,25 @@ def get(self, req):
 
             # Validating each param.
             board_uuid = UUID(board_uuid, version=4)
-            Board.objects.get(uuid=board_uuid)
             index = int(index)
             amount = int(amount)
         except:
```

Comment:
You should be catching exceptions you're interested in instead of one giant failsafe